test/ipc_devdax_prov.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (C) 2024 Intel Corporation
+#
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+#!/bin/bash
+
+set -e
+
+# port should be a number from the range <1024, 65535>
+PORT=$(( 1024 + ( $$ % ( 65535 - 1024 ))))
+
+UMF_LOG_VAL="level:debug;flush:debug;output:stderr;pid:yes"
+
+echo "Starting ipc_os_prov_shm CONSUMER on port $PORT ..."
```

Comment:
did you mean `Starting ipc_devdax_prov...` (same below)

Reply:
@lukaszstolarczuk Please verify and resolve.

Reply:
@lukaszstolarczuk ping